### ISSUE TITLE
fix mandatory metadata fields

### ISSUE THREAD

**Issue by jfrer** [subugoe/tido#150]
In GitLab by @krvoigt on Feb 4, 2021, 15:03

**Comment by jfrer** [comment]
In GitLab by @krvoigt on Feb 4, 2021, 15:03

added to epic subugoe&12

**Comment by jfrer** [comment]
In GitLab by @mweidling on Feb 5, 2021, 08:38

@krvoigt Please find my first draft attached.

[visualization_textapi.odt](https://gitlab.gwdg.de/subugoe/emo/tido/uploads/27a7a13c763324e395eb50d71fd4cac7/visualization_textapi.odt)

**Comment by jfrer** [comment]
In GitLab by @mweidling on Feb 5, 2021, 08:57

@krvoigt Could you give me feedback, please?

**Comment by jfrer** [comment]
In GitLab by @ghost on Feb 5, 2021, 17:04

@mweidling I talked to @krvoigt briefly. we agreed to start implementing it according to your draft (no worries, if there are some changes afterwards).

status quo: I implemented a new mixin, reflecting the detailed specs (data structure)

TODO:
- implement the logical part; e.g. compare the specs defined with the data Tido receives
- render the appropriate components
- configurability
